modules/katello_lifecycle_environment.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2017, Andrew Kofink <ajkofink@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: katello_lifecycle_environment
+short_description: Create and Manage Katello lifecycle environments
+description:
+    - Create and Manage Katello lifecycle environments
+author: "Andrew Kofink (@akofink)"
+requirements:
+    - "nailgun >= 0.28.0"
+    - "python >= 2.6"
+options:
+    server_url:
+        description:
+            - URL of Foreman server
+        required: true
+    username:
+        description:
+            - Username on Foreman server
+        required: true
+    password:
+        description:
+            - Password for user accessing Foreman server
+        required: true
+    verify_ssl:
+        description:
+            - Verify SSL of the Foreman server
+        default: false
+    name:
+        description:
+            - Name of the lifecycle environment
+        required: true
+    label:
+        description:
+            - Label of the lifecycle environment. This field cannot be updated.
+    description:
+        description:
+            - Description of the lifecycle environment
+    organization:
+        description:
+            - Organization name that the lifecycle environment is in
+        required: true
+    prior:
+        description:
+            - Name of the parent lifecycle environment
+    present:
+        description:
+            - Whether the lifecycle environment should be present or absent on the server
+        required: true
+'''
+
+EXAMPLES = '''
+- name: "Add a production lifecycle environment"
```

Comment:
So I was going to reply "because they need to run on the local machine, either by setting `hosts: localhost` (as in your demo playbook) or `delegate_to: localhost` (see ansible upstream)". But actually upstream does not document that for *all* the modules, example with delegate: http://docs.ansible.com/ansible/latest/azure_module.html#examples, so I am kinda torn on this now. Thanks ;)